python/lsst/meas/deblender/deblend.py

```diff
@@ -207,7 +207,7 @@ def addSchemaKeys(self, schema):
             'deblend_hasStrayFlux', type='Flag',
             doc=('This source was assigned some stray flux'))
 
-        self.log.logdebug('Added keys to schema: %s' % ", ".join(str(x) for x in (
+        self.log.trace('Added keys to schema: %s', ", ".join(str(x) for x in (
                     self.nChildKey, self.psfKey, self.psfCenterKey, self.psfFluxKey,
```

Comment:
It was discussed previously [here](https://github.com/lsst/meas_deblender/commit/6428a77262cdc9a9fe39e17da8a6bca9735b1c77) (and noted in the commit message).   For this logger named "meas_deblender.baseline", two tracing levels are currently used in master: a less verbose `log.log(-9, ...)` and a more verbose `log.logdebug(...)`. So the former maps to `log.debug()` and the latter maps to `log.trace()` in the conversion.